db/validate.js

```diff
@@ -0,0 +1,10 @@
+exports.courseDetails = (course) => {
+  var start_m = Date.parse(course.startdate)
+  var end_m = Date.parse(course.enddate)
+  var name_m = course.coursename.trim()
+  var currentDateArr = Date().split(' ')
+  var newCurrentDate = currentDateArr[3] + '/' + (new Date().getMonth() + 1) + '/' + currentDateArr[2]
```

Comment:
Always capture any logic in its own function. What is this line doing? Give it a function name which says precisely that. What is `currentDateArr[3]` and `currentDateArr[2]`? Give them function names which say precisely that.

Prefer functions to variables, especially in Javascript or any other language with function pointers. Even prefer them in languages which do not have function pointers. 😉